python/src/aiconfig/eval/custom_eval/examples/travel/travel_parametrized.aiconfig.json

```diff
@@ -12,24 +12,21 @@
       "gpt-4": {
         "model": "gpt-4",
         "max_tokens": 3000,
-        "system_prompt": "You are an expert travel coordinator with exquisite taste."
+        "system_prompt": "You are an expert travel coordinator with exquisite taste. You are an exremely concise, brief communicator."
       }
     },
     "default_model": "gpt-3.5-turbo"
   },
   "prompts": [
     {
       "name": "get_activities",
-      "input": "{{the_query}}"
+      "input": "Tell me 3 fun attractions related to {{the_query}} to do in NYC."
```

Comment:
The test prompt "food" seems super vague. I get that it makes sense now with this prompt, but I think it would be a bit easier if you said something like:

```
Tell me 3 fun attractions to do related to {{the_query}} in NYC."
```

And the prompts could be:
- different kinds of food
- iconic midtown skyscrapers